database/redis/latest.Dockerfile

```
@@ -0,0 +1,11 @@
+# https://github.com/redis/docker-library-redis/blob/e5650da99bb377b2ed4f9f1ef993ff24729b1c16/7.4/debian/Dockerfile
+# https://hub.docker.com/_/redis/tags
+FROM redis:latest
+
+LABEL org.opencontainers.image.description DESCRIPTION="Redis latest"
+LABEL org.opencontainers.image.source=https://github.com/lecaoquochung/docker-images/blob/main/ubuntu/latest.Dockerfile
+LABEL org.opencontainers.image.licenses=MIT
+
+# Update package index and install dependencies
+RUN apt-get update && \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)